### DISCUSSION TITLE
点歌 GFlowNet（Dr. Yoshua Bengio的最近的一作paper，开创性工作）

### DISCUSSION THREAD

**Discussion by AGIArchitect** [mli/paper-reading#126] [General]
https://arxiv.org/abs/2111.09266
https://github.com/bengioe/gflownet
Dr. Yoshua Bengio的一作paper，想看看沐神的理解。谢谢。

**Comment by jasperhyp** [comment]
资瓷资瓷，后来那篇70页实在读不下去了...
